safe/definitions/default_settings.py

```diff
@@ -23,6 +24,8 @@
     'ISO19115_EMAIL': 'info@inasafe.org',
     'ISO19115_LICENSE': 'Free use with accreditation',
 
+    'currency': idr['key'],
```

Comment:
IDR has the biggest number compared the other :D 

Or should we ask the user what currency they want to use after installing InaSAFE? Or is it overkill :D